lang/po/Firmware_sv.po

```diff
@@ -778,992 +778,992 @@ msgstr ""
 
 # MSG_TIMEOUT c=12
 #: messages.c:157
 msgid "Timeout"
-msgstr ""
+msgstr "Timeout"
 
 # MSG_X_CORRECTION c=13
 #: ultralcd.cpp:5086
 msgid "X-correct:"
-msgstr ""
+msgstr "X-korrektion:"
 
 # MSG_BED_SKEW_OFFSET_DETECTION_PERFECT c=20 r=8
 #: ultralcd.cpp:3737
 msgid "XYZ calibration ok. X/Y axes are perpendicular. Congratulations!"
-msgstr ""
+msgstr "XYZ-kalibrering ok. X/Y-axlarna ar vinkelräta. Grattis!"
 
 # MSG_BED_SKEW_OFFSET_DETECTION_WARNING_FRONT_BOTH_FAR c=20 r=8
 #: ultralcd.cpp:3721
 msgid "XYZ calibration compromised. Front calibration points not reachable."
-msgstr ""
+msgstr "XYZ-kalibrering komprometterad. Främre kalibreringspunkter kan inte nas."
 
 # MSG_BED_SKEW_OFFSET_DETECTION_WARNING_FRONT_RIGHT_FAR c=20 r=8
 #: ultralcd.cpp:3724
 msgid "XYZ calibration compromised. Right front calibration point not reachable."
-msgstr ""
+msgstr "XYZ-kalibrering komprometterad. Höger främre kalibreringspunkter kan inte nas."
 
 # MSG_LOAD_ALL c=17
 #: ultralcd.cpp:6167
 msgid "Load all"
-msgstr ""
+msgstr "Ladda alla"
 
 # MSG_BED_SKEW_OFFSET_DETECTION_POINT_NOT_FOUND c=20 r=6
 #: ultralcd.cpp:3703
 msgid "XYZ calibration failed. Bed calibration point was not found."
-msgstr ""
+msgstr "XYZ-kalibrering misslyckades. Bäddkalibreringspunkterna hittades inte."
 
 # MSG_BED_SKEW_OFFSET_DETECTION_FAILED_FRONT_BOTH_FAR c=20 r=6
 #: ultralcd.cpp:3709
 msgid "XYZ calibration failed. Front calibration points not reachable."
-msgstr ""
+msgstr "XYZ-kalibrering misslyckades. Främre kalibreringspunkter kan inte nas."
 
 # MSG_BED_SKEW_OFFSET_DETECTION_FAILED_FRONT_RIGHT_FAR c=20 r=6
 #: ultralcd.cpp:3712
 msgid "XYZ calibration failed. Right front calibration point not reachable."
-msgstr ""
+msgstr "XYZ-kalibrering misslyckades. Höger främre kalibreringspunkt kan inte nas."
 
 # MSG_Y_DIST_FROM_MIN c=20
 #: ultralcd.cpp:2842
 msgid "Y distance from min"
-msgstr ""
+msgstr "Y avstand fran min"
 
 # MSG_WIZARD_V2_CAL_2 c=20 r=12
 #: ultralcd.cpp:4850
 msgid "The printer will start printing a zig-zag line. Rotate the knob until you reach the optimal height. Check the pictures in the handbook (Calibration chapter)."
-msgstr ""
+msgstr "Skrivaren börjar skriva ut en sicksacklinje. Vrid vredet tills du nar optimal höjd. Kontrollera med bilderna i handboken (Kalibreringskapitlet)."
 
 # MSG_FIL_FAILED c=20 r=5
 #: ultralcd.cpp:7336
 msgid "Verification failed, remove the filament and try again."
-msgstr ""
+msgstr "Verifieringen misslyckades, ta bort filamentet och försök igen."
 
 # MSG_Y_CORRECTION c=13
 #: ultralcd.cpp:5087
 msgid "Y-correct:"
-msgstr ""
+msgstr "Y-korrektion:"
 
 # MSG_OFF c=3
 #: messages.c:125
 msgid "Off"
-msgstr ""
+msgstr "Av"
 
 # MSG_ON c=3
 #: messages.c:126
 msgid "On"
-msgstr ""
+msgstr "Pa"
 
 # MSG_BACK c=18
 #: messages.c:64
 msgid "Back"
-msgstr ""
+msgstr "Tillbaka"
 
 # MSG_CHECKS c=18
 #: ultralcd.cpp:5641
 msgid "Checks"
-msgstr ""
+msgstr "Kontrollerar"
```

Comment:
I think this is about "some checks" and not that it "checks" it now. @3d-gussner please